src/components/Modals/FiatRamps/views/Manager.tsx

```diff
@@ -88,14 +90,17 @@ const ManagerRouter: React.FC<ManagerRouterProps> = ({ fiatRampProvider }) => {
         if (supportsBTC(wallet) && bitcoinChainAdapter) {
           setBtcAddress(await bitcoinChainAdapter.getAddress(payload))
         }
+        if (supportsBTC(wallet) && dogecoinChainAdapter) {
```

Comment:
This works because native wallet works with UTXOs, thus works with doge, but do we want to be explicit and add more `_supportsXYZ` for other UTXOs in hdwallet? This is not obvious from looking at this diff that it is indeed a correct behavior and not a typo.

Reply:
yeah @asamere i have same concerns but can't recall if we talked about it - what happens when say Portis returns true for `supportsBTC` but doesn't support doge?

Reply:
supportsBTC in HDWallet means supportsUTXO, since these chains are literally clones and HDWallet is only concerned with the offline portions. I do agree it's confusing without the tribal knowledge piece.